docs/crio.8.md

```diff
@@ -125,6 +126,17 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--config, -c**="": Path to configuration file (default: /etc/crio/crio.conf)
 
+**--config-dir, -d**="": Path to the configuration drop-in directory.
+    This directory will be recursively iterated and each file gets applied
+    to the configuration in their processing order. This means that a
+    configuration file named '00-default' has a lower priority than a file
+    '01-my-overwrite'.
+    The global config file, either provided via '--config,-c' or per default in
+    /etc/crio/crio.conf, has always a lower priority than those inside the
+    '--config-dir,-d'.
+    Beside that, the provided command line parameters still have the highest
+    configuration priority. (default: /etc/crio/crio.conf.d)
```

Comment:
I'm not sure what this last bit is trying to explain.

Reply:
Changed it to:
```
	Beside that, provided command line parameters still have a higher priority
	than any configuration file.
```
I hope this makes more sense.